### PR TITLE
add restricted wallet and geoblocking error messages

### PR DESCRIPTION
**Pull request by mikemag-dev**
[TRCL-2642 : Show error state for when rate limited by indexer when calling screen API \(iOS\)](https://linear.app/dydx/issue/TRCL-2642/show-error-state-for-when-rate-limited-by-indexer-when-calling-screen)
[TRCL-2622 : Show full screen geoblock when receiving abacus error \(iOS\)](https://linear.app/dydx/issue/TRCL-2622/show-full-screen-geoblock-when-receiving-abacus-error-ios)
[TRCL-2624 : Call screen API when any wallet \(new or existing\) connects to FE \(iOS\)](https://linear.app/dydx/issue/TRCL-2624/call-screen-api-when-any-wallet-new-or-existing-connects-to-fe-ios)

### CODE REVIEW

**Comment by mikemag-dev** [comment]
pending legal's reply on copy, [thread](https://dydx-team.slack.com/archives/C05K1ACJ8MB/p1694108742843719)

File: config/localization/en/app.json [hunk 1]
```diff
@@ -633,9 +633,17 @@
...
+      "WALLET_RESTRICTED_ERROR_TITLE": "Wallet Restricted",
+      "WALLET_RESTRICTED_ERROR_MESSAGE": "Because this address appears to be a resident of, or trading from, a jurisdiction that violates our terms of use, or has engaged in activity that violates our terms of use, this address has been blocked.",
+      "WALLET_RESTRICTED_WITHDRAWAL_TRANSFER_DESTINATION_ERROR_MESSAGE": "Because the destination address appears to be a resident of, or trading from, a jurisdiction that violates our terms of use, or has engaged in activity that violates our terms of use, the destination address has been blocked and this transaction cannot be completed.",
+      "WALLET_RESTRICTED_WITHDRAWAL_TRANSFER_ORIGINATION_ERROR_MESSAGE": "Because the origination address appears to be a resident of, or trading from, a jurisdiction that violates our terms of use, or has engaged in activity that violates our terms of use, the origination address has been blocked and this transaction cannot be completed.",
+      "RATE_LIMIT_REACHED_ERROR_TITLE": "Rate Limit Reached",
+      "RATE_LIMIT_REACHED_ERROR_MESSAGE": "Rate limited reached for this IP address. Please try again later.",
```
Comment (jaredvu) [inline config/localization/en/app.json:641]: Could we move out of `APP.ONBOARDING` object into `ERRORS.ONBOARDING` object?
The two `RATE_LIMIT` strings could go into `ERRORS.GENERAL`

File: config/localization/en/app.json [hunk 2]
```diff
@@ -633,9 +633,17 @@
...
       "ABOUT_WALLETS": "About wallets",
       "ACKNOWLEDGE_TERMS": "Acknowledge terms",
       "API_KEYS_NOT_FOUND": "API Key not found",
+      "REGION_NOT_PERMITTED_TITLE": "dYdX Unavailable",
+      "REGION_NOT_PERMITTED_SUBTITLE": "Because you appear to be a resident of, or trading from, a jurisdiction that violates our terms of use, or have engaged in activity that violates our terms of use, you have been blocked. You may withdraw your funds from the protocol at any time.",
```
Comment (jaredvu) [inline config/localization/en/app.json:646]: `ERRORS.ONBOARDING`

**Review by jaredvu — APPROVED** [review]

**Comment by jaredvu** [comment]
Requires version bump

**Pull request merged**